src/packages/objectmanager/objectmanager.js

```diff
@@ -9,11 +9,12 @@ export class ObjectManager {
      * @returns ObjectManager
      */
     constructor() {
-        this.objects = {}
+        this.objects = new Map()
```

Comment:
Bin mir unsicher, ob man hier ne Map verwenden sollte. Wäre ein Array nicht besser? Dort könnte man z.B. in-place sortieren.

Reply:
die werden bereits in-place sortiert (https://github.com/joonlabs/fox.js/blob/dev/src/packages/objectmanager/objectmanager.js#L51)